examples/examples/opentelemetry.rs

```diff
@@ -15,9 +15,9 @@ fn expensive_work() -> &'static str {
 }
 
 fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
-    let (tracer, _uninstall) = opentelemetry_jaeger::new_pipeline()
+    let tracer = opentelemetry_jaeger::new_pipeline()
         .with_service_name("report_example")
-        .install()?;
+        .install_simple()?;
```

Comment:
nit, take it or leave it: i'm not currently looking at the `opentelemetry` docs, but based on the function name `install_simple`, I get the sense that there are other `install_xxx` methods? it might be worth a comment in the example explaining what this one does, in contrast to the others?

Reply:
Good idea. Added a brief comment about simple vs batch installation methods.